libs/entitlement/qa/entitlement-positive-check/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -0,0 +1,2 @@
+elasticsearch.qa.entitlement.positive:
```

Comment:
This is going to become cumbersome, duplicating to the non-modular case. Could we do some build magic to make the non-modular one a copy of this yaml file with the module name replaced?

Reply:
We can, but I think I'd like to do that separately as I need more time for "gradle magic" :)